src/domain/session/room/RoomViewModel.js

```diff
@@ -197,18 +197,80 @@ export class RoomViewModel extends ViewModel {
         }
     }
     
+    async _getMessageInformations (message) {
+        let msgtype = "m.text";
+        if (message.startsWith("/")) {
+            let command=message.substring(1).split(" ");
+            switch (command[0]) {
+                case "me":
+                   message = message.substring(4).trim();
+                   msgtype = "m.emote";
+                   break;
+               case "join":
+                   if (command.length==2) {
+                       let roomname = message.substring(5).trim();
+                       try {
+                           await this._options.client.session.joinRoom(roomname);
```

Comment:
After joining the room, we should switch over to the new room:
```suggestion
                           const internalId = await this._options.client.session.joinRoom(roomName);
                           this.navigation.push("room", internalId);
```

Reply:
Also wanted to mention that we use camel casing for variables, so prefer `roomName` over `roomname`.